packages/blade/src/components/Dropdown/docs/stories.ts

```diff
@@ -539,8 +539,6 @@ const WithControlledSelectStory = `
             <ActionList>
               <ActionListItem title="Mumbai" value="mumbai" />
               <ActionListItem title="Bangalore" value="bangalore" />
-              <ActionListItem title="Pune" value="pune" />
-              <ActionListItem title="Chennai" value="chennai" />
```

Comment:
unrelated change?

Reply:
@saurabhdaware There is not enough space to render all options which flips the position of the dropdown in controlled select [story](https://61c19ee8d3d282003ac1d81c-fanhpizsbk.chromatic.com/?path=/docs/components-dropdown-dropdown--page).
<img width="1103" alt="image" src="https://github.com/razorpay/blade/assets/48921037/98aebd3f-4760-4a27-8773-db9e08ef7ad7">

Reply:
oh got it. btw, when there is no enough space on top and bottom both, I think we should prefer opening it in bottom only right?

Reply:
Yeah, I have made the change. Please review once